README.md

```diff
@@ -191,7 +191,7 @@ Restart IDE.
 - Enable <kbd>EnvFile</kbd> checkbox
 - Click on <kbd>+</kbd> to add a file
 
-![Read from file](./resources/example.png)
+![Read from file](./example.png)
```

Comment:
Is there is a reason why we dropping resources dir? I'm not feeling comfortable flooding the root of the project with all the resources such as screenshots and whatnot without a good reason for that.

Reply:
Previously everything in the `resources` directory including `example.png` made its way into the plugin jar and consequently into the runtime classpath. That is why I decided to place it in the root project directory next to the `README.md` file. Should I put it inside the other directory for images?
```
$ unzip -l EnvFile.jar
Archive:  EnvFile.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
       52  04-18-2018 20:49   META-INF/MANIFEST.MF
        0  04-18-2018 20:49   META-INF/
     2445  04-18-2018 20:49   META-INF/plugin.xml
      270  04-18-2018 20:49   META-INF/envfile-rubymine.xml
      244  04-18-2018 20:49   META-INF/envfie-idea.xml
      249  04-18-2018 20:49   META-INF/envfile-pycharm.xml
    98930  04-18-2018 20:49   example.png
        0  04-18-2018 20:49   net/
...
---------                     -------
   193125                     45 files
```

Reply:
Ugh, good catch! We definitely don't need images out there.